GoogleForCreators/web-stories-wp
# Floating menu: Replace media button

## Context



## Summary
Makes "Replace media" button functional.


## Relevant Technical Choices



## To-do



## User-facing changes
<img width="428" alt="Screenshot 2022-03-01 at 14 32 37" src="https://user-images.githubusercontent.com/3294597/156169700-49051a61-f956-4963-8dd4-3c70e9a2839c.png">



## Testing Instructions






This PR can be tested by following these steps:

Make sure the Floating menu experiment is enabled first.

1. Add an image.
2. Verify "Replace media" button is displayed in the floating menu
3. Verify it works as expected
4. Verify the same for video.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?
Adds new tracking:
```
      trackEvent('floating_menu', {
        name: 'replace_media',
        element: type,
      });
```


### Does this PR have a legal-related impact?



## Checklist




---



Fixes #10635
